### PR TITLE
Create huseyin.json

### PR DESCRIPTION
**Pull request by Huseyin-Cinar**
To make our job easier, please spend time to review your application before submitting. 

- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.

### CODE REVIEW

File: domains/huseyin.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "description": "Hüseyin Çınar Web Sitesi",
+  "repo": "https://github.com/Huseyin-Cinar/huseyin.dev",
+  "owner": {
+    "username": "Huseyin-Cinar",
+    "email": "gslicocuk8080@gmail.com"
+  }
```
Comment (Huseyin-Cinar) [inline domains/huseyin.json:7]: a

**Comment by Huseyin-Cinar** [comment]
Birleştirmem neden onaylanmadı?

**Review by MaskDuck — CHANGES_REQUESTED** [review]
this website wasn't written in english, right? can you elaborate on what does the content in this website means? and also can you fix the `description` field to use english?

thanks.

**Comment by Huseyin-Cinar** [comment]
sure

**Comment by Huseyin-Cinar** [comment]
This web site redirecting to futuree.netlify.app adress. Want you futuree.netlify.app's means?

**Comment by Huseyin-Cinar** [comment]
@MaskDuck

**Comment by MaskDuck** [comment]
yes

**Comment by Huseyin-Cinar** [comment]
This website, from many parts of the world; was established to gather people with different purposes such as software, coding, programming, domain acpuisition, and opening a website on a single website.

**Review by MaskDuck — DISMISSED** [review]

**Comment by MaskDuck** [comment]
acceptable ig idk

**Comment by MaskDuck** [comment]
will have to look into this further sorry

**Review by TheBotlyNoob — DISMISSED** [review]
:+1:

**Review by MaskDuck — APPROVED** [review]
we're sorry for late reply :(

**Pull request merged**